cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackendImpl.java

```diff
@@ -124,7 +124,7 @@ public void createCollection(String dbName, String collectionName, long dataExpi
                 .append("_id.attrName", 1)
                 .append("_id.resolution", 1)
                 .append("_id.origin", 1);
-            options = new IndexOptions();
+            options = new IndexOptions().name("index");
```

Comment:
```suggestion
            options = new IndexOptions().name("cygnus_agg_optimization_index");
```

In addition, the index created a L136 should have a different name, e.g: "cygnus_agg_expiration_index"